charon/Cargo.toml

```diff
@@ -70,6 +70,7 @@ tracing = { version = "0.1", features = [ "max_level_trace", "release_max_level_
 which = "6.0.1"
 
 hax-frontend-exporter = { git = "https://github.com/hacspec/hax", branch = "main", optional = true }
+# hax-frontend-exporter = { git = "https://github.com/Nadrieril/hax", branch = "update-rustc", optional = true }
```

Comment:
Remove this

Reply:
I left it on purpose because I use it often